### PR TITLE
test: add email sending actions

### PR DESCRIPTION
**Pull request by hellyzh**

### CODE REVIEW

File: .github/workflows/graph-toolkit-released.yml [hunk 1]
```diff
@@ -26,4 +26,19 @@ jobs:
           subject="Microsoft Graph Toolkit v${{ github.event.client_payload.version }} Released"
 
           echo "to=$emails" >> $GITHUB_OUTPUT
-          echo "subject=$subject" >> $GITHUB_OUTPUT
\ No newline at end of file
+          echo "subject=$subject" >> $GITHUB_OUTPUT
```
Comment (wenytang-ms) [inline .github/workflows/graph-toolkit-released.yml:29]: why not use the send-email-report action?

File: .github/workflows/teamsjs-released.yml [hunk 2]
```diff
@@ -43,4 +43,19 @@ jobs:
           subject="Teams JS SDK v${{ github.event.client_payload.version }} Released"
 
           echo "to=$emails" >> $GITHUB_OUTPUT
-          echo "subject=$subject" >> $GITHUB_OUTPUT
\ No newline at end of file
+          echo "subject=$subject" >> $GITHUB_OUTPUT
+
+      - name: Send E-mail Notifications
```
Comment (wenytang-ms) [inline .github/workflows/teamsjs-released.yml:48]: same here

File: .github/workflows/graph-toolkit-released.yml (hunk 1 above)
Comment (hellyzh) [reply .github/workflows/graph-toolkit-released.yml:29]: Their commands are different, no double quotation marks for them. It is html not only string using "body": ${{ steps.detail.outputs.result }},

Comment (hellyzh) [reply .github/workflows/graph-toolkit-released.yml:29]: The original stopping email PR as reference: https://github.com/OfficeDev/TeamsFx/pull/9481/files#diff-64fade0467319bab9b796ca00c34fcb10fcd65d42e7f461aa6cc688898762de9L56

Comment (hellyzh) [reply .github/workflows/graph-toolkit-released.yml:29]: updated to use send-email-report action

File: .github/workflows/teamsjs-released.yml (hunk 2 above)
Comment (hellyzh) [reply .github/workflows/teamsjs-released.yml:51]: updated to use send-email-report action

**Review by wenytang-ms — APPROVED** [review]

**Pull request merged**